docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -335,7 +335,7 @@ Now there are two AccountUpdates:
   - The parent corresponds to the `add()` method call
   - The child corresponds to the `this.incrementCalls()` call that the parent makes.
 
-One update is the child because it was called from the parent, which also implies that the parent has the child included as part of its proof. To learn more about parent/child account updates, see [Payment to zkApp example](/zkapps/o1js/interact-with-mina#payment-to-zkapp-example).
+One update is the child because it was called from the parent, which also implies that the parent has the child included as part of its proof. To learn more about parent/child account updates, see [Payment to zkApp example](/zkapps/o1js/interact-with-mina#signing-transactions-and-explicit-account-updates).
```

Comment:
```suggestion
One update is the child because it was called from the parent, which also implies that the parent has the child included as part of its proof. To learn more about parent/child account updates, see [Signing transactions and explicit account updates](/zkapps/o1js/interact-with-mina#signing-transactions-and-explicit-account-updates) in the Payment to zkApp example.
```

Reply:
We want the link text to match the destination, so if we link to the heading for Signing transactions and explicit account updates we change the link text to match.

Reply:
@barriebyron thanks, noted